coopr-docs/docs/source/installation/packages.rst

```diff
@@ -138,20 +138,25 @@ Yum
 ---
 To install each of the Coopr components locally from a Yum package:
 
-.. parsed-literal::
-  $ sudo yum localinstall coopr-server-\ |version|\ .el6.x86_64.rpm
-  $ sudo yum localinstall coopr-provisioner-\ |version|\ .el6.x86_64.rpm
-  $ sudo yum localinstall coopr-ui-\ |version|\ .el6.x86_64.rpm
+.. container:: highlight
 
+  .. parsed-literal::
+   
+    |$| sudo yum localinstall coopr-server-|version|.el6.x86_64.rpm
```

Comment:
"$" renders as a "$", but highlighted in orange. See the staged example. It's hard to see the difference, because now it matches all the other examples.